js/viz/funnel.d.ts

```diff
@@ -57,13 +58,14 @@ export {
     Palette,
     PaletteExtensionMode,
     TextOverflow,
+    ShiftLabelOverlap,
     WordWrap,
 };
 
 /** @public */
 export type FunnelAlgorithm = 'dynamicHeight' | 'dynamicSlope';
 /** @public */
-export type FunnelLabelOverlap = 'hide' | 'none' | 'shift';
+export type FunnelLabelOverlap = ShiftLabelOverlap;
```

Comment:
I think we should just deprecate `FunnelLabelOverlap` and recommend using of `ShiftLabelOverlap`